controllers/TuitController.ts

```diff
@@ -46,6 +49,8 @@ export default class TuitController implements ITuitController {
       );
       app.put("/api/tuits/:tid", TuitController.tuitController.updateTuit);
       app.delete("/api/tuits/:tid", TuitController.tuitController.deleteTuit);
+      app.get("/api/tuits/:tid/versions", TuitController.tuitController.getVersions);
+      app.put("/api/edit", TuitController.tuitController.editTuit);
```

Comment:
Looking at the URLs of other tuit related endpoints, I think a better endpoint for editTuit would be something like
`/api/tuits/:tid/edit`